Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.2.3"
+ruby "2.3.0"
```

Comment:
Well, we're supporting Rails 4 as well. So, we should be able to support older versions of ruby if we can.